server/handlers.go

```diff
@@ -473,6 +455,20 @@ func (s *Server) handleConnectorCallback(w http.ResponseWriter, r *http.Request)
 // finalizeLogin associates the user's identity with the current AuthRequest, then returns
 // the approval page's path.
 func (s *Server) finalizeLogin(identity connector.Identity, authReq storage.AuthRequest, conn connector.Connector) (string, error) {
+
+	/*
+	 * Giant Swarm custom code to inject connector prefix in the group names, so it enables us
```

Comment:
We move this here from the callback handler to the login finalizing step. This way it will be called by all types of conntectors.